tests/testthat/test-onesample.R

```diff
@@ -36,4 +38,22 @@ test_that("Von Mises Assumption Check table results match", {
                                       "WindDirection", 0.01, 0.164, 5.50228611491871, "July", 0.0644792434618744,
                                       "WindDirection", 0.01, 0.128, 1.7314268317444, "September",
                                       0.0642929504742415, "WindDirection"))
+})
+
+
+test_that("Von Mises Assumption Check catches too concentrated data", {
+  options <- analysisOptions("CircularStatisticsOneSampleTests")
+  options$.meta <- list(splitby = list(shouldEncode = TRUE), variables = list(shouldEncode = TRUE))
+  options$alphaRao <- "0.01"
+  options$alphaVonMises <- "0.01"
+  options$period <- 360
+  options$periodGroup <- "custom"
+  options$variables <- "contWide"
+  options$vonMisesCheck <- TRUE
+  
+  set.seed(1)
+  results <- runAnalysis("CircularStatisticsOneSampleTests", "test.csv", options)
+  errorMessage <- results[["results"]][["vonMisesCheckTable"]][["error"]][["errorMessage"]]
+  testthat::expect_identical(errorMessage, 
+                             gettextf("Estimated %s is infinite, could not compute results. Your data is too concentrated to calculate the assumption check.", "\u03BA"))
```

Comment:
I don't really know how to fix this properly, it should just work the way you did it. We could manipulate the unicode with stringi/ stringr, but I think it's more appropriate to do
```r
if (.Platform$OS.type == "windows") {
  testthat::expect_true(grepl("Estimated \\S is infinite, could not compute results. Your data is too concentrated to calculate the assumption check.", errorMessage))
} else {
  testthat::expect_identical(errorMessage, 
                             gettextf("Estimated %s is infinite, could not compute results. Your data is too concentrated to calculate the assumption check.", "\u03BA"))
}
```
Here `\\S` is a catch-all for a single non-whitespace character.

Reply:
Thanks! Let's hope that the test will now pass